Assets/Scripts/CopilotMechanics/MushroomFriendActive.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MushroomFriendActive : CopilotActiveMechanic
+{
+    public GameObject HealingBubblePrefab;
+    private HealingBubble healingBubbleScript;
+    private GameObject healingBubble;
```

Comment:
Can I get a comment on this variable? I see that it's the bubble that got instantiated but at a glance it was hard to know that. Just a simple comment helps with that :)